activesupport/test/message_verifier_test.rb

```diff
@@ -24,6 +24,7 @@ def test_valid_message
     data, hash = @verifier.generate(@data).split("--")
     assert !@verifier.valid_message?(nil)
     assert !@verifier.valid_message?("")
+    assert !@verifier.valid_message?("\xff")
```

Comment:
Yes, I head my mind on the exception. Thanks!

The comment should now be there.